docs/en/setup/backend/backend-storage.md

```diff
@@ -26,10 +26,15 @@ storage:
     user: sa
 ```
 
-## ElasticSearch 6
-Active ElasticSearch 6 as storage, set storage provider to **elasticsearch**.
+## ElasticSearch
+Active ElasticSearch 6 as storage, set storage provider to **elasticsearch**, active ElasticSearch 7 as storage, set storage provider to **elasticsearch7**.
```

Comment:
These two should be point 1 and 2. Such as 

* Active ElasticSearch 6 as storage, set storage provider to **elasticsearch**
* Active ElasticSearch 7 as storage, set storage provider to **elasticsearch7**

In this review page, https://github.com/apache/skywalking/blob/dbe634b47e18b8b3b5c0f7ea98b62778afd705bb/docs/en/setup/backend/backend-storage.md#elasticsearch, it is not very clear.